systemtest/src/main/java/io/strimzi/systemtest/utils/StUtils.java

```diff
@@ -202,12 +206,13 @@ public static boolean checkLogForJSONFormat(Map<String, String> pods, String con
         boolean isJSON = false;
 
         for (String podName : pods.keySet()) {
-            String logs = kubeClient().logs(podName, containerName).replaceFirst("([^{]+)", "");
-            try {
-                new JsonObject(logs);
+            String log = cmdKubeClient().exec(true, false, "logs", podName, "-c", containerName, "--tail=100").out();
+            log = log.replaceAll("[\n|\r]", "");
+            jsonMatcher = JSON_PATTERN.matcher(log);
```

Comment:
Again, just to clarify, I assume it's fine that `JSON_PATTERN` doesn't understand nested `{}`, and that you're not expecting that in the `log`?

Reply:
If I remember correctly, at least one of the loggers has some nested JSON objects. So it should not rely on the log messages being always flat.